src/generator/protocol/operations.ts

```diff
@@ -127,40 +137,44 @@ function formatParamValue(param: Parameter, imports: ImportManager): string {
     case SchemaType.DateTime:
     case SchemaType.Duration:
     case SchemaType.UnixTime:
-      return `${param.language.go!.name}.String()`;
+      if (param.required !== true) {
+        // remove the dereference (bit of a hack but this is the only case where it's necessary)
+        paramName = paramName.substr(1);
```

Comment:
If we want it to be a little less hacky we could add a check to see if the dereference * exists on the string and then perform the substr func. Not something I would insist on, but I did something similar when parsing the response structs to remove the trailing }.

Reply:
I was thinking about it...I think it might be safer to add the check. Just in case we were ever to pass something that doesnt include the deref

Reply:
Good point, added.